torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -714,6 +713,9 @@ def _is_packable_convolution(match):
                 return False
         is_transposed = conv_node.args[-3]
         if is_transposed:
+            # TODO: Support dynamic shape case for MKLDNN conv transpose.
+            if free_symbols(input_size):
+                return False
```

Comment:
Do we have a test to cover this unsupported case?

Reply:
Yes, we have a test to cover it.